simplesamlphp_auth.module

```diff
@@ -680,7 +680,7 @@ function _simplesamlphp_auth_generate_block_text() {
     . '<br />' . l(t('Log Out'), 'user/logout') . '</p>';
   }
   else {
-    $block_content .= '<p>' . config_get('simplesamlphp_auth.settings', 'login_link_display_name') . '</p>';
+    $block_content .= '<p>' . l(config_get('simplesamlphp_auth.settings', 'login_link_display_name'), 'saml_login') . '</p>';  }
```

Comment:
@irinaz 
At a glance, the `   }` on the end of this line looks suspicious. Is that intentional?

Reply:
Oops :(  sorry, fixed this typo.